tests/unit/expire.tcl

```diff
@@ -261,100 +275,286 @@ start_server {tags {"expire"}} {
+            $primary setex foo5 100 bar
+            $primary psetex foo6 100000 bar
+            # EXPIRE-family commands
+            $primary set foo7 bar
+            $primary expire foo7 100
+            $primary set foo8 bar
+            $primary pexpire foo8 100000
+            $primary set foo9 bar
+            $primary expireat foo9 [expr [clock seconds]+100]
+            $primary set foo10 bar
+            $primary pexpireat foo10 [expr [clock milliseconds]+100000]
+            # GETEX commands
+            $primary set foo11 bar
+            $primary getex foo11 ex 100
+            $primary set foo12 bar
+            $primary getex foo12 px 100000
+            $primary set foo13 bar
+            $primary getex foo13 exat [expr [clock seconds]+100]
+            $primary set foo14 bar
+            $primary getex foo14 pxat [expr [clock milliseconds]+100000]
+            # RESTORE commands
+            $primary set foo15 bar
+            set encoded [$primary dump foo15]
+            $primary restore foo16 100000 $encoded
+            $primary restore foo17 [expr [clock milliseconds]+100000] $encoded absttl
+
+            # Wait for replica to get the keys and TTLs
+            assert {[$primary wait 1 1000] == 1}
+
+            # Verify absolute TTLs are identical on primary and replica
+            # This is because TTLs are always replicated as absolute values
+            assert {[$primary expiretime foo1 ptime] eq [$replica expiretime foo1 ptime]}
+            assert {[$primary expiretime foo2 ptime] eq [$replica expiretime foo2 ptime]}
+            assert {[$primary expiretime foo3 ptime] eq [$replica expiretime foo3 ptime]}
+            assert {[$primary expiretime foo4 ptime] eq [$replica expiretime foo4 ptime]}
+            assert {[$primary expiretime foo5 ptime] eq [$replica expiretime foo5 ptime]}
+            assert {[$primary expiretime foo6 ptime] eq [$replica expiretime foo6 ptime]}
+            assert {[$primary expiretime foo7 ptime] eq [$replica expiretime foo7 ptime]}
+            assert {[$primary expiretime foo8 ptime] eq [$replica expiretime foo8 ptime]}
+            assert {[$primary expiretime foo9 ptime] eq [$replica expiretime foo9 ptime]}
+            assert {[$primary expiretime foo10 ptime] eq [$replica expiretime foo10 ptime]}
+            assert {[$primary expiretime foo11 ptime] eq [$replica expiretime foo11 ptime]}
+            assert {[$primary expiretime foo12 ptime] eq [$replica expiretime foo12 ptime]}
+            assert {[$primary expiretime foo13 ptime] eq [$replica expiretime foo13 ptime]}
+            assert {[$primary expiretime foo14 ptime] eq [$replica expiretime foo14 ptime]}
+            assert {[$primary expiretime foo15 ptime] eq [$replica expiretime foo15 ptime]}
+            assert {[$primary expiretime foo16 ptime] eq [$replica expiretime foo16 ptime]}
+            assert {[$primary expiretime foo17 ptime] eq [$replica expiretime foo17 ptime]}
```

Comment:
let's use `assert_equal` rather than `assert`.
the benefit is that when the assertion fails, the message contains the values.
also, let's convert this big block to a loop using KEYS, rather than a line per key.

Reply:
Done